views/homepage/index.html

```diff
@@ -9,6 +9,14 @@
     <link rel="stylesheet" href="/resources/stylesheets/userbar.css">
     <link rel="stylesheet" href="/homepage/stylesheets/style.css">
     <title>Thimble by Mozilla - An online code editor for learners &amp; educators.</title>
+
+    {% if GA_ACCOUNT %}
```

Comment:
These are defined globally here https://github.com/mozilla/thimble.webmaker.org/blob/bramble/app.js#L125-L126, so should already exist, no?